test/runtests.jl

```diff
@@ -64,6 +64,9 @@ include("utils/example_tasks.jl")
         include("solvers/test_truncated_cg.jl")
         include("solvers/test_trust_regions.jl")
     end
-    include("MOI_wrapper.jl")
+    if VERSION >= v"1.9"
```

Comment:
We currently try to support all versions down to 1.6, but this seems to make It necessary to have 1.9?

Reply:
Are the other extensions of Manopt working for v1.6 ? I wanted to be consistent with the other extensions but didn't see how you made them work for v1.6

Reply:
Sure, with requires, see https://github.com/JuliaManifolds/Manopt.jl/blob/c4bc49dc6c028c8c6bf994d3e053c9c8c1098a11/src/Manopt.jl#L182-L200

Reply:
Ah, easy then, done :)